### PR TITLE
[DX-846] Dashboard-5.2.3-draft

### PR DESCRIPTION
**Pull request by andyo-tyk**
Preview: https://deploy-preview-3631--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/release-notes/version-5.2/

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 1]
```diff
@@ -14,6 +14,56 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+
+#### Upgrade instructions
+If you are using a 5.2.x version, we advise you to upgrade ASAP to this latest release. If you are on an older version, you should skip 5.2.0 and upgrade directly to this release.
+
+#### Release Highlights
+This release primarily focuses on bug fixes. 
+For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.2.3">}}) below.
+
+#### Downloads
+- [Docker image to pull](https://hub.docker.com/layers/tykio/tyk-dashboard/v5.2.3/images/TBC?context=explore)
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
- [Docker image to pull](https://hub.docker.com/layers/tykio/tyk-dashboard/v5.2.3/images/sha256-7d61ed3ee3f03ff0e2f91be71a9113b90ef6637b1cef1f30d4c3e04ead09fa6a?context=explore)
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 2]
```diff
@@ -14,6 +14,56 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+    <li>CVE-2023-44487 - to mitigate this vulnerability, users are advised to disable HTTP/2 at this time by setting `enable_http2` to `false`</li>
+    <li>CVE-2023-39325</li>
+  ​</ul>
+
+<li><details><summary>DoS risk from pagination query parameter</summary> Fixed a security issue where passing a non-integer value as the pagination query parameter `p` to the `/api/logs` endpoint could lead to an out-of-memory scenario as the Dashboard would attempt to retrieve all logs in the system. Tyk will now return an `HTTP 400 Bad Request` response if a non-integer value is provided. This fix mitigates the risk of accidentally or deliberately causing Tyk Dashboard to stop responding.</details></li>
+
+#### Fixed
+
+<ul><li><details><summary>Unable to resize OPA editor in Tyk Dashboard</summary> Fixed an issue where the OPA editor in the Tyk Dashboard was not resizable. The fix ensures the floating OPA editor is now resizable and the resizing operation is smooth, improving user experience.</details></li>
+​
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 3]
```diff
@@ -14,6 +14,56 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+<li><details><summary> User Search not working unless you enter the full email address</summary> Fixed an issue where the User Search was not working unless the full email address was entered. The fix restores the functionality of showing suggestions for names as they are typed in, improving user experience and search efficiency.</details></li>
+​
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 4]
```diff
@@ -14,6 +14,56 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+    <li>CVE-2023-45283</li>
+    <li>CVE-2023-44487 - to mitigate this vulnerability, users are advised to disable HTTP/2 at this time by setting `enable_http2` to `false`</li>
+    <li>CVE-2023-39325</li>
+  ​</ul>
+
+<li><details><summary>DoS risk from pagination query parameter</summary> Fixed a security issue where passing a non-integer value as the pagination query parameter `p` to the `/api/logs` endpoint could lead to an out-of-memory scenario as the Dashboard would attempt to retrieve all logs in the system. Tyk will now return an `HTTP 400 Bad Request` response if a non-integer value is provided. This fix mitigates the risk of accidentally or deliberately causing Tyk Dashboard to stop responding.</details></li>
+
+#### Fixed
+
+<ul><li><details><summary>Unable to resize OPA editor in Tyk Dashboard</summary> Fixed an issue where the OPA editor in the Tyk Dashboard was not resizable. The fix ensures the floating OPA editor is now resizable and the resizing operation is smooth, improving user experience.</details></li>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
<ul><li><details><summary>Unable to resize OPA editor in Tyk Dashboard</summary> Fixed an issue where the [OPA editor]({{< ref "tyk-dashboard/open-policy-agent#using-the-open-policy-agent-in-the-dashboard" >}}) was not resizable. The fix ensures the floating OPA editor is now resizable and the resizing operation is smooth, improving user experience.</details></li>
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 5]
```diff
@@ -14,6 +14,56 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+​
+<li><details><summary> User Search not working unless you enter the full email address</summary> Fixed an issue where the User Search was not working unless the full email address was entered. The fix restores the functionality of showing suggestions for names as they are typed in, improving user experience and search efficiency.</details></li>
+​
+<li><details><summary>Dashboard 4.1.0+ cannot retrieve certificates from downrev gateways</summary> Fixed an issue where Dashboard 4.1.0+ was unable to retrieve certificates from a Tyk Gateway with a version lower than 4.1.0. This was due to a change made in the 4.1 versions to the way certificate details are retrieved in dashboard; in the newer versions, we can view more detail of the certificates. Now you can use Tyk Dashboard with any version of the Tyk Gateway and still retrieve and view certificate details; the fix ensures smooth staged upgrades and prevents potential issues for customers who have weeks or months between upgrading components.</details></li>
+​
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 6]
```diff
@@ -14,6 +14,56 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+<li><details><summary>Authentication Mode changes after updating the "Protocol" for an API</summary> Fixed an issue with the Classic API Designer where the 'Authentication Mode' was changing to 'Authentication Token' after updating the 'Protocol' for an API in the Dashboard UI. This fix ensures the 'Authentication Mode' remains consistent after making changes to the 'Protocol'.</details></li>
+​
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 7]
```diff
@@ -14,6 +14,56 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+<li><details><summary>Unable to configure external OAuth flow using Raw API editor</summary> Fixed an issue in the Classic API Designer where the 'use_standard_auth' value was constantly reverting to 'true' when editing an API with an external OAuth flow. This fix ensures the 'use_standard_auth' value remains consistent, enabling the use of external OAuth via the Raw API editor.</details></li>
+​
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
```

**Review by andyo-tyk — COMMENTED** [review]
Removed blank lines

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 8]
```diff
@@ -14,6 +14,56 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+  ​</ul>
+
+<li><details><summary>DoS risk from pagination query parameter</summary> Fixed a security issue where passing a non-integer value as the pagination query parameter `p` to the `/api/logs` endpoint could lead to an out-of-memory scenario as the Dashboard would attempt to retrieve all logs in the system. Tyk will now return an `HTTP 400 Bad Request` response if a non-integer value is provided. This fix mitigates the risk of accidentally or deliberately causing Tyk Dashboard to stop responding.</details></li>
+
+#### Fixed
+
+<ul><li><details><summary>Unable to resize OPA editor in Tyk Dashboard</summary> Fixed an issue where the OPA editor in the Tyk Dashboard was not resizable. The fix ensures the floating OPA editor is now resizable and the resizing operation is smooth, improving user experience.</details></li>
+​
+<li><details><summary> User Search not working unless you enter the full email address</summary> Fixed an issue where the User Search was not working unless the full email address was entered. The fix restores the functionality of showing suggestions for names as they are typed in, improving user experience and search efficiency.</details></li>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
<li><details><summary> User Search not working unless you enter the full email address</summary> Fixed an issue where the [User Search]({{< ref "basic-config-and-security/security/dashboard/search-users" >}}) was not working unless the full email address was entered. The fix restores the functionality of showing suggestions for names as they are typed in, improving user experience and search efficiency.</details></li>
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 9]
```diff
@@ -14,6 +14,56 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+​
+<li><details><summary>Dashboard 4.1.0+ cannot retrieve certificates from downrev gateways</summary> Fixed an issue where Dashboard 4.1.0+ was unable to retrieve certificates from a Tyk Gateway with a version lower than 4.1.0. This was due to a change made in the 4.1 versions to the way certificate details are retrieved in dashboard; in the newer versions, we can view more detail of the certificates. Now you can use Tyk Dashboard with any version of the Tyk Gateway and still retrieve and view certificate details; the fix ensures smooth staged upgrades and prevents potential issues for customers who have weeks or months between upgrading components.</details></li>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
<li><details><summary>Dashboard 4.1.0+ cannot retrieve certificates from downrev gateways</summary> Fixed an issue where Dashboard 4.1.0+ was unable to retrieve certificates from a Tyk Gateway with a version lower than 4.1.0. This was due to a change made in the 4.1 versions relating to the way certificate details are retrieved in dashboard; in the newer versions, we can view more detail of the certificates. Now you can use Tyk Dashboard with any version of the Tyk Gateway and still retrieve and view certificate details; the fix ensures smooth staged upgrades and prevents potential issues for customers who have weeks or months between upgrading components.</details></li>
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 10]
```diff
@@ -14,6 +14,51 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+  ​</ul>
+
+<li><details><summary>DoS risk from pagination query parameter</summary> Fixed a security issue where passing a non-integer value as the pagination query parameter `p` to the `/api/logs` endpoint could lead to an out-of-memory scenario as the Dashboard would attempt to retrieve all logs in the system. Tyk will now return an `HTTP 400 Bad Request` response if a non-integer value is provided. This fix mitigates the risk of accidentally or deliberately causing Tyk Dashboard to stop responding.</details></li>
+
+#### Fixed
+
+<ul><li><details><summary>Unable to resize OPA editor in Tyk Dashboard</summary> Fixed an issue where the OPA editor in the Tyk Dashboard was not resizable. The fix ensures the floating OPA editor is now resizable and the resizing operation is smooth, improving user experience.</details></li>
+<li><details><summary> User Search not working unless you enter the full email address</summary> Fixed an issue where the User Search was not working unless the full email address was entered. The fix restores the functionality of showing suggestions for names as they are typed in, improving user experience and search efficiency.</details></li>
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
<li><details><summary>User Search not working unless you enter the full email address</summary> Fixed an issue where the User Search was not working unless the full email address was entered. The fix restores the functionality of showing suggestions for names as they are typed in, improving user experience and search efficiency.</details></li>
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 11]
```diff
@@ -14,6 +14,51 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+
+#### Security
+  ​
+The following High Priority CVEs are known to exist in this release:
+ <ul>
+    <li>CVE-2023-45283</li>
+    <li>CVE-2023-44487 - to mitigate this vulnerability, users are advised to disable HTTP/2 at this time by setting `enable_http2` to `false`</li>
+    <li>CVE-2023-39325</li>
+  ​</ul>
+
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 12]
```diff
@@ -14,6 +14,56 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+​
+<li><details><summary>Dashboard 4.1.0+ cannot retrieve certificates from downrev gateways</summary> Fixed an issue where Dashboard 4.1.0+ was unable to retrieve certificates from a Tyk Gateway with a version lower than 4.1.0. This was due to a change made in the 4.1 versions to the way certificate details are retrieved in dashboard; in the newer versions, we can view more detail of the certificates. Now you can use Tyk Dashboard with any version of the Tyk Gateway and still retrieve and view certificate details; the fix ensures smooth staged upgrades and prevents potential issues for customers who have weeks or months between upgrading components.</details></li>
+​
+<li><details><summary>Authentication Mode changes after updating the "Protocol" for an API</summary> Fixed an issue with the Classic API Designer where the 'Authentication Mode' was changing to 'Authentication Token' after updating the 'Protocol' for an API in the Dashboard UI. This fix ensures the 'Authentication Mode' remains consistent after making changes to the 'Protocol'.</details></li>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]: Is this the link to [Authentication Mode](https://tyk.io/docs/basic-config-and-security/security/authentication-authorization/oauth-2-0/#1-set-the-authentication-mode) content?

```suggestion
<li><details><summary>Authentication Mode changes after updating the "Protocol" for an API</summary> Fixed an issue with the Classic API Designer where the [Authentication Mode({{< ref "basic-config-and-security/security/authentication-authorization/oauth-2-0#1-set-the-authentication-mode" >}}) was changing to 'Authentication Token' after updating the 'Protocol' for an API in the Dashboard UI. This fix ensures the 'Authentication Mode' remains consistent after making changes to the 'Protocol'.</details></li>
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 13]
```diff
@@ -14,6 +14,56 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+​
+<li><details><summary>Unable to configure external OAuth flow using Raw API editor</summary> Fixed an issue in the Classic API Designer where the 'use_standard_auth' value was constantly reverting to 'true' when editing an API with an external OAuth flow. This fix ensures the 'use_standard_auth' value remains consistent, enabling the use of external OAuth via the Raw API editor.</details></li>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
<li><details><summary>Unable to configure external OAuth flow using Raw API editor</summary> Fixed an issue in the Classic API Designer where the 'use_standard_auth' value was constantly reverting to 'true' when editing an API with an [external OAuth flow]({{< ref "basic-config-and-security/security/authentication-authorization/ext-oauth-middleware" >}})). This fix ensures the 'use_standard_auth' value remains consistent, enabling the use of external OAuth via the Raw API editor.</details></li>
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 14]
```diff
@@ -14,6 +14,56 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+​
+<li><details><summary>If the GraphQL subscription upstream dies, the UI is unaware of the reconnection event</summary> Fixed an issue with failed GraphQL subscriptions between the upstream and the Dashboard. When an upstream subscription was disconnected and later reconnected, the UI did not update to reflect the reconnection, preventing the seamless consumption of messages. Now the Dashboard can continue consuming messages after upstream reconnects.</details></li>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
<li><details><summary>If the GraphQL subscription upstream disconnects, the UI is unaware of the reconnection event</summary> Fixed an issue with failed GraphQL subscriptions between the upstream and the Dashboard. When an upstream subscription was disconnected and later reconnected, the UI did not update to reflect the reconnection, preventing the seamless consumption of messages. Now the Dashboard can continue consuming messages after upstream reconnects.</details></li>
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 15]
```diff
@@ -14,6 +14,56 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+#### Changelog {#Changelog-v5.2.3}
+
+#### Security
+  ​
+The following High Priority CVEs are known to exist in this release:
+ <ul>
+    <li>CVE-2023-45283</li>
+    <li>CVE-2023-44487 - to mitigate this vulnerability, users are advised to disable HTTP/2 at this time by setting `enable_http2` to `false`</li>
+    <li>CVE-2023-39325</li>
+  ​</ul>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]: Remove unfixed external dependency tracked CVEs

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Hi Andy, PR looks good to me, I've added minor suggestions, including links

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md [hunk 16]
```diff
@@ -14,6 +14,50 @@ Minor releases are supported until our next minor comes out. There is no 5.3 sch
...
+<li><details><summary>DoS risk from pagination query parameter</summary> Fixed a security issue where passing a non-integer value as the pagination query parameter `p` to the `/api/logs` endpoint could lead to an out-of-memory scenario as the Dashboard would attempt to retrieve all logs in the system. Tyk will now return an `HTTP 400 Bad Request` response if a non-integer value is provided. This fix mitigates the risk of accidentally or deliberately causing Tyk Dashboard to stop responding.</details></li>
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
<ul><li><details><summary>DoS risk from pagination query parameter</summary> Fixed a security issue where passing a non-integer value as the pagination query parameter `p` to the `/api/logs` endpoint could lead to an out-of-memory scenario as the Dashboard would attempt to retrieve all logs in the system. Tyk will now return an `HTTP 400 Bad Request` response if a non-integer value is provided. This fix mitigates the risk of accidentally or deliberately causing Tyk Dashboard to stop responding.</details></li></ul>
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md (hunk 12 above)
Comment (andyo-tyk) [reply tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]: Not really, no - I don't think there is a single link in the docs for that control. Probably better with just a generic link to AuthN - such as https://tyk.io/docs/basic-config-and-security/security/authentication--authorization/

Comment (andyo-tyk) [reply tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md]:
```suggestion
<li><details><summary>Authentication Mode changes after changing API Protocol in API Designer</summary> Fixed an issue in the Tyk Classic API Designer where if you changed the protocol for an API (for example from HTTP to HTTPS) then the authentication mechanism would be automatically set to Authentication Token.</details></li>
```

**Review by dcs3spp — APPROVED** [review]
Thanks Andy I have reviewed changes and updated the PR for the other patch releases to include consistent changelog formatting. @agata-wit if you could peer review and then we can merge and release

**Pull request merged**